route/route.go

```diff
@@ -606,16 +611,59 @@ func (router *Router) processOTLPRequest(
 				Environment: environment,
 				SampleRate:  uint(ev.SampleRate),
 				Timestamp:   ev.Timestamp,
-				Data:        types.NewPayload(router.Config, ev.Attributes),
+				Data:        payload,
```

Comment:
not a big deal but is this change necessary?

Reply:
It's _unnecessary_ :). I removed it